components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1057,9 +1058,6 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         const ws = await this.workspaceService.getWorkspace(user.id, workspaceId);
         await this.guardAccess({ kind: "workspace", subject: ws }, "delete");
 
-        // for good measure, try and stop running instances
-        await this.internalStopWorkspace(ctx, user.id, ws, "deleted via API");
```

Comment:
@geropl I deleted this here because it happens as part of deleteWorkspace below.

Reply:
Only difference I see is the "delete via API" is replaced by "deleted via WorkspaceService". But not sure how we used that anyway. :+1: